TypeCobol/Compiler/SqlCodeElements/Statement/CommitStatement.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TypeCobol.Compiler.CodeElements;
+using TypeCobol.Compiler.Scanner;
+using TypeCobol.Compiler.SqlScanner;
+
+namespace TypeCobol.Compiler.SqlCodeElements.Statement
+{
+    /// <summary>
+    /// SQL COMMIT Statement Code Element.
+    /// </summary>
+    public class CommitStatement : SqlStatementElement
+    {
+        public CommitStatement() : base(CodeElementType.CommitStatement, StatementType.CommitStatement)
+        {
+        }
+
+        public override bool VisitCodeElement(IASTVisitor astVisitor)
+        {
+            return base.VisitCodeElement(astVisitor);
+        }
```

Comment:
Useless override, it should be removed.

Reply:
done in [7f2afb7](https://github.com/TypeCobolTeam/TypeCobol/pull/1928/commits/7f2afb778747d8f26ff0ce751781c2ab8845aa78)